Visual Studio/Realm.vcxproj

```diff
@@ -363,7 +363,8 @@
       <PrecompiledHeader>
       </PrecompiledHeader>
       <DisableSpecificWarnings>4355;4996</DisableSpecificWarnings>

```

Comment:
You forgot to answer this one, even removing these disabled warnings should probably be done in another PR.